pragma/file.go

```diff
@@ -49,6 +49,7 @@ func (f *File) extractPragmas() {
 	reader := bufio.NewReader(strings.NewReader(f.Content))
 
 	currentLine := 0
```

Comment:
should be current line index no?
we do `lineNum`.

Reply:
No I meant we should call it, `currentLineNum`.

Reply:
Got it, will rename.